katas/langtonant/richard/module1.js

```diff
@@ -1,133 +0,0 @@
-
```

Comment:
Richard did a refactor, but still, maybe this should not be removed to prevent merge conflicts